.github/workflows/ci.yml

```diff
@@ -1,50 +1,55 @@
-# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
-# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+name: Continuous integration
+on: 
+  push:
+  pull_request:
+    branches: [master]
 
-name: Node.js CI
+jobs:
+  tests:
+    runs-on: ubuntu-latest
 
-on:
-    push:
-        branches: [master]
-    pull_request:
-        branches: [master]
+    strategy:
+      matrix:
+        node-version: [12.x, 14.x, 16.x]
+        cucumber-version: [7.x, 8.x]
 
-jobs:
-    build:
-        runs-on: ubuntu-latest
-
-        strategy:
-            matrix:
-                node-version: [12.x, 14.x, 16.x]
-                cucumber-version: [7.x, 8.x]
-
-        steps:
-            - uses: actions/checkout@v2
-            - name: Use Node.js ${{ matrix.node-version }}
-              uses: actions/setup-node@v1
-              with:
-                  node-version: ${{ matrix.node-version }}
-            - run: yarn add -P @cucumber/cucumber@${{ matrix.cucumber-version }}
```

Comment:
Unless I'm wrong, it seems like you removed this step, which means we do not test against all cucumber versions anymore but only the one in our dev dependencies, which the minimum of the two (7.x).

You need to add it back so our tests are executed on both versions.

Reply:
Done